app/serializers/credentials_serializer.rb

```diff
@@ -3,40 +3,40 @@ def initialize(user: , application:)
     @user, @application = user, application
   end
 
-  def serialize
-    {
-      user: serialized_user,
-      profile: serialized_profile,
-      roles: serialized_roles,
-    }
-  end
-
-  def to_json(opts = {})
-    serialize.to_json opts
+  def as_json(opts = {})
+    serialize
   end
 
   private
 
   attr_reader :user, :application
 
+  def serialize
+    {
+      "user" => serialized_user,
+      "profile" => serialized_profile,
+      "roles" => serialized_roles,
+    }
+  end
+
   def serialized_user
     {
-      email: user.email,
+      "email" => user.email,
     }
   end
 
   def serialized_profile
     {
-      email: profile.email,
-      name:  profile.name,
-      telephone: profile.tel,
-      mobile: profile.mobile,
-      address: {
-        full_address: profile.address,
-        postcode: profile.postcode,
+      "email" => profile.email,
```

Comment:
Rails docs doesn't explicitly state `as_json` must return a hash with string keys:
http://api.rubyonrails.org/classes/ActiveModel/Serializers/JSON.html#method-i-as_json

If internally Rails calls `to_json()` to create JSON string, then symbol keys would still result in the same JSON output:

```ruby
{ 'foo': 'bar' }.to_json
=> "{\"foo\":\"bar\"}"

{ foo: 'bar' }.to_json
=> "{\"foo\":\"bar\"}"
```